DeusEx/Classes/DXRFixup.uc

```diff
@@ -303,6 +338,11 @@ function HongKong_FirstEntry()
                     case "LumpathPissed":
                         Trigger(a).bTriggerOnceOnly = False;
                         break;
+                    case "Keypad3":
+                        if( a.Event == 'elevator_door' && HackableDevices(a) != None ) {
+                            HackableDevices(a).hackStrength = 0;
```

Comment:
not sure if I should make this 0.1 instead, but 0.6 is the default and that means 6 multitools if you're unskilled just to go back to the previous area! it's really bad if you don't have the helipad set as a fixedConn in DXREntraceRando